api/src/routers/meals.js

```diff
@@ -0,0 +1,93 @@
+import "dotenv/config";
+import express from "express";
+import knex from "../database_client.js";
+
+const mealsRouter = express.Router();
+
+// Future Meals
+mealsRouter.get("/future-meals", async (req, res) => {
+  try {
+    const futureMealsQuery =
+      process.env.DB_CLIENT === "mysql2"
+        ? "SELECT * FROM meal WHERE `when` > NOW() ;"
+        : "SHOW TABLES;";
+    const tables = await knex.raw(futureMealsQuery);
```

Comment:
The `knex.raw()` call returns an array, with two elements:
1.  An array, containing all the resulting rows as objects
2. The schema of the table

You should not return the schema, meaning that you should only return the first value of the `tables` array. Or, an even better way to do this would be to write somwthing like: 
```suggestion
    const [tables, schema] = await knex.raw(futureMealsQuery);
```
And then `tables` will have the value you want to return.

Also, why would we call this variable `tables`? It should probably be called either `rows`, or more specifically `meals`, right?

Reply:
Thanks for the constructive feedback, now it is more clear for me, done!